Project.toml

```
@@ -1,6 +1,6 @@
 name = "RData"
 uuid = "df47a6cb-8c03-5eed-afd8-b6050d6c41da"
-version = "0.7.1"
+version = "0.7.3"
```

it should be:
```suggestion
version = "0.7.2"
```
I think (otherwise Registrator will disallow auto-merge)

---

good catch!